### PR TITLE
bundle: rearrange embed.FS setup

### PR DESCRIPTION
**Pull request by srenatus**
No functional changes, I hope!

### CODE REVIEW

File: bundle/bundle.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package bundle
+
+import (
+	"embed"
+)
+
+// Note: this will bundle the tests as well, but since that has negligible impact on the size of the binary,
+// it's preferable to filter them out from the bundle than to e.g. create a separate directory for tests
+//
+//go:embed *
```
Comment (srenatus) [inline bundle/bundle.go:10]: 💭 Probably means that _this_ file makes it into the bundle, too. But it should be ignored...?

**Review by anderseknert — APPROVED** [review]
That's an interesting approach, and not one I had considered :)
Having a `.go` file in the bundle may not be koscher, but yeah, it should just get ignored.

File: bundle/bundle.go (hunk 1 above)
Comment (anderseknert) [reply bundle/bundle.go:10]: https://pkg.go.dev/embed#hdr-Directives

> If a pattern names a directory, all files in the subtree rooted at that directory are embedded (recursively), except that files with names beginning with ‘.’ or ‘_’ are excluded. 

Does that mean the file could be named `_bundle.go`, or would that not work with Go? 🤔

File: bundle/bundle.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+var Bundle embed.FS
```
Comment (anderseknert) [inline bundle/bundle.go:11]: >  Error: ST1022: comment on exported var Bundle should be of the form "Bundle ..." (stylecheck)

Adding an ignore on package level should be fine here.

File: internal/io/io.go [hunk 3]
```diff
@@ -94,7 +94,7 @@ func CloseFileIgnore(file *os.File) {
 }
 
 func ExcludeTestFilter() filter.LoaderFilter {
-	return func(abspath string, info files.FileInfo, depth int) bool {
+	return func(_ string, info files.FileInfo, _ int) bool {
```
Comment (srenatus) [inline internal/io/io.go:97]: Sorry, unrelated change. I had just happened to look into how the bundle is loaded...

File: bundle/bundle.go (hunk 1 above)
Comment (srenatus) [reply bundle/bundle.go:10]: I was hoping that `*` explicitly included files starting with `.` -- we need it for `.manifest`.

**Comment by srenatus** [comment]
Tests fail, back to draft. I'm looking into it.

File: bundle/bundle.go (hunk 1 above)
Comment (anderseknert) [reply bundle/bundle.go:10]: Oh right, you'll need to use `all` for that, I think

Comment (srenatus) [reply bundle/bundle.go:10]: ✔️ Double-checked: `.manifest` is there, and `bundle.go` is, too.

Comment (anderseknert) [reply bundle/bundle.go:10]: Yay!
